### PR TITLE
[RNMobile][Monorepo] Fix license check errors

### PR DESCRIPTION
**Pull request by ceyhun**
## Description

This is a part of the migration of gutenberg-mobile to the gutenberg repo.

Fixes: https://github.com/WordPress/gutenberg/issues/19861

## How has this been tested?

- `npm run check-licenses` should complete without errors
- `npm run lint` as well

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/react-native-editor/__device-tests__/helpers/utils.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import childProcess from 'child_process';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Tug) [inline packages/react-native-editor/__device-tests__/helpers/utils.js:5]: Not sure how this is supposed to be handled on gutenberg, we can't have dev dependencies inside packages but there is this lint rule as well.
It seems there would be other packages where that would be a problem if they were modified today.

**Review by Tug — APPROVED** [review]
LGTM 👍

File: packages/react-native-editor/__device-tests__/helpers/utils.js (hunk 1 above)
Comment (ceyhun) [reply packages/react-native-editor/__device-tests__/helpers/utils.js:5]: Tests could be a special case where we import dev dependencies. What do you think about excluding this folder altogether from that rule?

Comment (Tug) [reply packages/react-native-editor/__device-tests__/helpers/utils.js:5]: I'm not sure, feels like I'm missing some context on why we have that kind of rules in the first place. Let's just keep it for now

**Pull request merged**